test/com/nutrons/nu17/MovingAverageTest.java

```diff
@@ -0,0 +1,18 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import lib.MovingAverage;
+
+public class MovingAverageTest {
+
+	@Test
+	public void movingAverageValuesTest() {
```

Comment:
One test is nowhere near enough. Your test could be passed by code that vapidly returns 3 or that includes only the first number in the average or that only divides by 3. I should be able to read your tests and be convinced that your code works.